app/api/config.ts

```diff
@@ -6,6 +6,7 @@ const {
   UPLOADS_FOLDER,
   CUSTOM_UPLOADS_FOLDER,
   TEMPORAL_FILES_FOLDER,
+  ACTIVITY_LOGS,
```

Comment:
@Zasa-san all the other variables for folders config have a ```_FOLDER``` at the end, can you change the name for the acritivy_logs var also ?